### ISSUE TITLE
CPU and Energy issue persists for me :( 

### ISSUE THREAD

**Issue by fabi94music** [kmikiy/SpotMenu#41]
Hi, 
I posted this earlier. 
I notices that SpotMenu is a little resource hungry on my system. 
As it was not the case on your system, I don't know what the reason is.
Could you have a look into that? If you need additional files (or reports) please let me know which ones and how to send you. 
It is not that huge of an issue but I wonder why the process on my system is so dominant. 
I didn't skip tracks, just listened to music track by track. 
<img width="806" alt="bildschirmfoto 2017-07-22 um 20 13 08" src="https://user-images.githubusercontent.com/29753374/28493632-3e53fa6a-6f1a-11e7-9075-6d9d3f15f198.png">

<img width="795" alt="bildschirmfoto 2017-07-22 um 20 13 53" src="https://user-images.githubusercontent.com/29753374/28493649-633dbbea-6f1a-11e7-8e52-c47e3c2007d9.png">
<img width="558" alt="bildschirmfoto 2017-07-22 um 21 38 08" src="https://user-images.githubusercontent.com/29753374/28494133-1a10dab8-6f26-11e7-8830-118bfd780bf7.png">

**Comment by kmikiy** [comment]
I updated the source code.
CPU performance should be a lot better now.
On baseline 2016 Macbook Pro: cpu should be around 0.4% and around 2% when you skip to the next song.
Energy impact should also be around 0.4 
[download](https://github.com/kmikiy/SpotMenu/releases/download/v1.5/SpotMenu.zip)
Let me know if it's better

**Comment by fabi94music** [comment]
Hey 😮 😃 
seem to be much better on my MacBook Pro as well! 
Really good job!

**Comment by kmikiy** [comment]
I'll close it for now, and see if I can optimize it a little more later on

**Comment by alexpersian** [comment]
Hello! Sorry to resurrect this issue, but with version 1.9 installed I've noticed similar CPU usage stats. What I've found is that this seems to depend on whether or not the bar display is scrolling.

With scrolling:
<img width="559" alt="screen shot 2019-02-13 at 2 12 37 pm" src="https://user-images.githubusercontent.com/5446451/52737225-ce59bc80-2f99-11e9-9f6a-01e60e1d5c9f.png">

W/o scrolling:
<img width="561" alt="screen shot 2019-02-13 at 2 13 08 pm" src="https://user-images.githubusercontent.com/5446451/52737240-d7e32480-2f99-11e9-9ff9-48ee4e7edfca.png">

This is on macOS Mojave 10.14.2
